versioned_docs/version-v0.75/node-operators/migration-guides/upgrade-node.md

```diff
@@ -55,9 +101,15 @@ We recommend checking all of the changes on your own. Follow the below instructi
   - data-node config
 3. Compare the new generated file in the temp location and the old file to see the differences.
 
-
 ## Upgrade your node
 To upgrade the network follow the [protocol upgrade documentation](../how-to/upgrade-network.md).
 
+## After you upgrade
+
+1. There is a new network parameter for `evmbridgeconfigs`. This value is hard-coded as it needs to be available when your node starts up.
```

Comment:
I am not sure if it is important but the network param is called: `blockchains.evmBridgeConfigs`